components/filter/filter.js

```diff
@@ -213,11 +214,13 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		}
 		return this._dimensions.map((dimension) => {
 			const builtDimension = this._buildDimension(dimension);
-			const dimensionDescription = `${dimension.text}. ${this.localize('components.filter.filterCountDescription', { number: dimension.appliedCount })}`;
-			return html`<d2l-menu-item text="${dimension.text}" description="${dimensionDescription}">
+			const dimensionDescription = `${dimension.text}.`;
```

Comment:
Minor: Probably don't need this variable anymore, we can just use `dimension.text` directly in the `menu-item`.